app/models/link.rb

```diff
@@ -1,4 +1,18 @@
 class Link < ApplicationRecord
   has_and_belongs_to_many :jobs
+  has_many :checks
+
   validates_presence_of :uri
+
+  def existing_check
+    checks
+      .where(ended_at.gt(24.hours.ago))
+      .first
+  end
```

Comment:
I imagine this will be configurable by input. Might be more idiomatic to do this with `("ended_at > ?", Time.now - input)`